docs/json/sonarr/cf/10bit.json

```diff
@@ -1,5 +1,6 @@
 {
   "trash_id": "b2550eb333d27b75833e25b8c2557b38",
+  "trash_score": "0",
```

Comment:
To my knowledge if a CF shouldn't get a score (0) then leave the score empty, Would need to ask @austinwbest if that's still the case,

Reply:
Spoke with Nit, you are correct have to remove the 0 scores